node/clients/checkout.ts

```diff
@@ -46,17 +46,25 @@ export class Checkout extends JanusClient {
   public addItem = (
     orderFormId: string,
     items: Array<Omit<OrderFormItemInput, 'uniqueId' | 'index' | 'options'>>,
-    salesChannel?: string,
-    allowOutdatedData?: string[]
-  ) =>
+    options?: {
+      salesChannel?: string,
+      allowOutdatedData?: string[],
+      splitItem?: boolean 
+    }) =>
     this.patch<CheckoutOrderForm>(
       this.routes.addItem(
         orderFormId,
-        this.getChannelQueryString(salesChannel)
+        this.getChannelQueryString(options?.salesChannel)
       ),
+      (options?.splitItem === undefined) ?
+      { 
+        orderItems: items,
+        allowedOutdatedData: options?.allowOutdatedData
+      } :
       {
         orderItems: items,
-        allowedOutdatedData: allowOutdatedData,
+        allowedOutdatedData: options?.allowOutdatedData,
+        noSplitItem: !(options.splitItem)
       },
```

Comment:
I think you can keep the object intact and add `noSplitItem` conditionally, just like that:
```typescript
{
	orderItems: items,
	allowedOutdatedData: options?.allowOutdatedData,
	...(options?.splitItem ? { noSplitItem: !(options.splitItem) } : {})
}
```

[Reference.](https://stackoverflow.com/a/55396398/7651928)